test/python/quantum_info/operators/test_measures.py

```diff
@@ -184,6 +186,17 @@ def test_diamond_norm(self, num_qubits):
         except cvxpy.SolverError:
             self.skipTest("CVXPY solver failed.")
 
+    def test_unitary_diamond_distance(self):
+        """Test the unitary_diamond_distance function"""
+        # Test unitary diamond distance for random unitaries
```

Comment:
This comment does not describe what the test is actually doing (testing RZGates for a specific set of angles). However, I agree that testing random unitaries is a good idea. Please add a test that does that, comparing the answer with the one obtained by the existing `diamond_norm` function.

Reply:
That sounds good. I will make the change. As a side note, the probability that the distance between two Harr random unitaries equals 2 increases exponentially with the number of qubits. Practically, testing random 3-qubit unitaries almost always returns a distance of 2. Additionally, it takes a while to run `diamond_norm` for 3 qubit channels (about 4 seconds). 

Would running the test for several random 1 and 2-qubit unitaries make for a better test?